### PR TITLE
SAK-29475 Switch to valid email address for demo.

### PR DESCRIPTION
**Pull request by buckett**
The sample demo users now have valid demo email address so you don’t have to change it.

### CODE REVIEW

**Comment by buckett** [comment]
Thanks to feedback from @jonespm use the standard example domain for education.

**Comment by ottenhoff** [comment]
Big +1 saves time for QA work

File: providers/sample/src/java/org/sakaiproject/provider/user/SampleUserDirectoryProvider.java [hunk 1]
```diff
@@ -276,7 +279,7 @@ public boolean findUserByEmail(UserEdit edit, String email)
 		if ((edit == null) || (email == null)) return false;
 
 		// assume a "@local.host"
```
Comment (juanjmerono) [inline providers/sample/src/java/org/sakaiproject/provider/user/SampleUserDirectoryProvider.java]: This comment should be changed/removed.
Anyway this should be merged even with this comment :smile:

File: providers/sample/src/java/org/sakaiproject/provider/user/SampleUserDirectoryProvider.java [hunk 2]
```diff
@@ -305,7 +308,7 @@ public Collection findUsersByEmail(String email, UserFactory factory)
 		UserEdit edit = factory.newUser();
 
 		// assume a "@local.host"
```
Comment (juanjmerono) [inline providers/sample/src/java/org/sakaiproject/provider/user/SampleUserDirectoryProvider.java]: This comment should be changed/removed.
Anyway this should be merged even with this comment :smile:

**Comment by buckett** [comment]
@juanjmerono Thanks, fixed.

**Pull request merged**